### DISCUSSION TITLE
Mekanism machines crafting

### DISCUSSION THREAD

**Discussion by NickCloudAT** [RS485/LogisticsPipes#1601] [Q&A] [answered]
Hey,

soo I use the newest Beta Build of Logistic Pipes (0.10.3.72) also tried with the latest stable (0.10.3.40) and the latest Mekanism version for 1.12.2.

What is my Issue?

I tried to autocraft Items in Mekanism machines (for this example I use a Metalluric Infuser to craft for example Enriched Alloy), but the crafting pipe is not able to pull out the result from the machine.
Machine Config: https://prnt.sc/vf6yga

I tried it like this cause I read it works on some forum. I also tried this on 1.7.10 and there it works without a problem.
Also tried a chasis pipe with a crafting module, does not work either.

Is there any way this works?

**Comment by Bernasss12** [comment, marked as answer]
So I don't know how you've setup the pipes, and at this point I wouldn't expect a response, but since I've recently gotten to play with both mods I can say that I managed to setup autocrafting with every machine so far.

As you probably noticed Mekanism machines can only either input or output to a single slot from each side of the machine, and that being so the setup I managed to get working with the Metallurgic Infuser was to have the Crafting Pipe/Chassis Pipe w/ Module on the output face, one satellite pipe on the "Extra" input and another one on the "Input" input. For this machine is specific you would need an Advanced Satellite upgrade for each recipe since it needs two satellites.

So in the end this seems to work, and Crafting Pipes/Modules can pull stuff from the Output of Mekanism machines as far as I can see. Again I don't know how you had your pipes setup so I can't say what was broken, maybe it was LP, maybe it was Mek but this doesn't seem to be a problem anymore.
